elements/modal.js

```diff
@@ -121,15 +122,17 @@ const prefix = css`
 module.exports = createWidget
 
 function createWidget () {
+  let isCopied = false
```

Comment:
That's what we're doing right? Or are you talking about the unidirectional rendering stuff being too complicated? - I think I definitely prefer explicit state and re-rendering over using a selector and some sort of "toggle class" method.

Would perhaps adding a method called `reset` and calling that when we reset stuff help?